component/core/src/main/java/org/exoplatform/social/core/jpa/search/ActivitySearchConnector.java

```diff
@@ -47,7 +47,7 @@ public class ActivitySearchConnector {
       "  \"query_string\":{" +
       "    \"fields\": [\"body\", \"posterName\"]," +
       "    \"default_operator\": \"AND\"," +
-      "    \"query\": \"@term@\"" +
+      "    \"query\": \"@term@"+ "~1" + "\"" +
```

Comment:
What if there are multiple words, please use `fuzziness` field instead as suggested before

```suggestion
      "    \"query\": \"@term@"+ "~1" + "\"," +
      "    \"fuzziness\": 1," +
      "    \"phrase_slop\": 1" +
```

See https://www.elastic.co/guide/en/elasticsearch/reference/7.17/query-dsl-query-string-query.html#query-string-top-level-params